### PR TITLE
Document enterprise feature for base comparison

### PR DESCRIPTION
**Pull request by nightfurys**

### CODE REVIEW

**Comment by nightfurys** [comment]
@zhill I have some todos here for links to the UI and ancestors pages. Also I'm not entirely sure about where the base comparison for vulnerabilities should land. I created a sparsely populated section with a giant todo for now but that can be easily changed. Let me know what you think

File: content/docs/overview/concepts/base_images/_index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: "Base Images"
+linkTitle: "Base Images"
+weight: 3
+---
+
+An image is considered base or ancestor if it's layers are a subset of another image. It is possible for an image to have more than one base image, wherein the image shares common layers with different base images to a varying degree. For instance consider an application image built on top of a webapp image that in turn is built on top of an scratch image. Both the webapp and scratch images are bases for the application. The webapp image is a maximal base since it has the most number of layers in common with the application while the scratch image is a minimal base since it has the least number of common layers           
+
```
Comment (nightfurys) [inline content/docs/overview/concepts/base_images/_index.md:8]: @zhill add a small para here for ancestor detection linking it to its own page. Feel free to modify/improve my description of base images as you see fit

File: content/docs/overview/concepts/base_images/checks.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+linkTitle: "Compare Policy Checks"
+weight: 1
+---
+
+This feature provides a mechanism to compare the policy checks for an image with those of a base image. You can read more about base images and how to find them [here]({{< ref "/docs/overview/concepts/base_images" >}}). Base comparison uses the same policy bundle and tag to evaluate both images to ensure a fair comparison. The API yields a response similar to the policy checks API with an additional element within each triggered gate check to indicate whether the result is inherited from the base image.
+
+### Usage
+
+This functionality is currently available via the Enterprise UI and API. Watch this space as we add base comparison support in other tools
+
```
Comment (nightfurys) [inline content/docs/overview/concepts/base_images/checks.md:12]: @katgurdak I've removed placeholders/inaccurate links for now but this is the "Enterprise UI" section should be added. Similar format at the notification pages

File: content/docs/overview/concepts/base_images/vulnerabilities.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+linkTitle: "Compare Vulnerabilities"
+weight: 2
+---
+
+This feature provides a mechanism to compare the security vulnerabilities detected in an image with those of a base image. You can read more about base images and how to find them [here]({{< ref "/docs/overview/concepts/base_images" >}}). The API yields a response similar to vulnerabilities API with an additional element within each result to indicate whether the result is inherited from the base image.
+
+### Usage
+
+This functionality is currently available via the Enterprise UI and API. Watch this space as we add base comparison support in other tools
+
```
Comment (nightfurys) [inline content/docs/overview/concepts/base_images/vulnerabilities.md:12]: @katgurdak I've removed placeholders/inaccurate links for now but this is the "Enterprise UI" section should be added. Similar format at the notification pages

**Comment by zhill** [comment]
I added my image ancestry doc, #212 as a draft. I'll look at how to combine this. I think I may just merge this and #210 as well and then update to make them all consistent rather than juggling a bunch of PRs in parallel.

**Comment by nightfurys** [comment]
> I think I may just merge this and #210 as well and then update to make them all consistent rather than juggling a bunch of PRs in parallel.
@zhill, I opened a separate ticket for the UI docs to track it better. Those docs might be coming in a little later and I don't want to lose the cross linking between UI and backend sections

**Pull request merged**